src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+          action="#"
+          method="post"
+          class="form"
+          data-qa="big"
+        >
+      <input
+          type="search"
+          name="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="form__input"
+          data-qa="keypress"
+        >
+      <img
+            class="form__img"
+            src="./images/Search.svg"
+            alt="search"
+          >
```

Comment:
Fix formatting according to this example
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/96409563/87b6c5ff-e32b-4e2d-9b01-6e21e64e85c9)

Reply:
Agree, do it